server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -528,23 +529,40 @@ private void registerAsyncMetrics(MeterRegistry registry) {
             )
         );
 
+        AtomicLong totalCoordinatingRequestsPrevious = new AtomicLong(0);
+        AtomicLong totalCoordinatingRejectionsPrevious = new AtomicLong(0);
```

Comment:
Is this our current monitoring strategy? Historically we had always reported forever incrementing counters. And relied on the end user to calculate diffs. For example, ES does not need to do a diff here. Report the totals, and the Kibana can perform the diff for calculating the ratio.

I'm not totally up to date on what peoples thoughts are.

Reply:
We can report some avg metrics like load or cpu usage. But for the most part we prefer to submit monotonically increasing metrics.

@volodk85 is there a reason that the "ratio" cannot be calculated in the aggregation instead?

Reply:
it is possible. reason behind implementation as an aggregated value was just simplicity (since we need it in one place only). After a second thought I simplified the implementation - right now it emits only raw counters which are required to calculate ratio rejected vs processed in kibana. Moreover this metrics (counters) can support graphs (should we need more than just graphs of spiking rejection ratio) 

Thank you